src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -621,3 +626,15 @@ extern "C" void GetStreamSizes(int32_t* header, int32_t* trailer, int32_t* maxim
         *maximumMessage = SSL3_RT_MAX_PLAIN_LENGTH;
     }
 }
+
+
+extern "C" bool SslAddExtraChainCert(SSL* ssl, X509* x509)
```

Comment:
We should not be using bool in our native shim signatures.  This should be int32_t, and we should explicitly return 0 or 1.